core/src/system_monitor_service.rs

```diff
@@ -21,7 +25,7 @@ const MS_PER_H: u64 = MS_PER_M * 60;
 const SAMPLE_INTERVAL_UDP_MS: u64 = 2 * MS_PER_S;
 const SAMPLE_INTERVAL_OS_NETWORK_LIMITS_MS: u64 = MS_PER_H;
 const SAMPLE_INTERVAL_MEM_MS: u64 = MS_PER_S;
-const SAMPLE_INTERVAL_CPU_MS: u64 = MS_PER_S;
+const SAMPLE_INTERVAL_CPU_MS: u64 = 10 * MS_PER_S;
```

Comment:
this should probably go in separately

@brennanwatt any concerns with bumping the cpu info metrics submission out to 10s?

Reply:
I'll do a separate PR for that. As I was writing the PR comment I realized that it was probably a separate thing :(